libnetdata/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.14)
+project(queue_test)
+
+set(CMAKE_C_STANDARD 99)
+# GoogleTest requires at least C++11
+set(CMAKE_CXX_STANDARD 11)
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+)
```

Comment:
We already had `cmocka` for our tests, is really necessary to add another tester? Why cannot we use `cmocka`?

Reply:
Another thing, I did not find references for these files in our autoconf files or cmake files.